Lib/Environment.cpp

```diff
@@ -127,9 +127,9 @@ bool Environment::timeLimitReached() const
  */
 int Environment::remainingTime() const
 {
-  // If time limit is set to 0 then assume we always have a minute left
+  // If time limit is set to 0 then assume we always have an hour left
```

Comment:
No problem with it, but why?

Reply:
Thinking of the use case with time limit set to 0 (in the master process) - which means "run forever". I want to run strategies scheduled with a time limit greater than 1 minute each...

Reply:
Makes sense... btw most users don't know about the time limit = 0 means forever behavior